### DISCUSSION TITLE
Should we use this Discussions Forum? (yes)

### DISCUSSION THREAD

**Discussion by salkinium** [modm-io/modm#693] [General]
I've enabled the GitHub Discussions Forum for this repo, for you to explore. I feel like opening issues for questions or discussions is not ideal and it probably scares people off that would otherwise just post a question.
At least for me it's probably better to discuss  in here and then formulate an issue or direct a PR.

Would you feel more comfortable using this forum than issues for questions you have?
cc @TomSaw @twasilczyk @mcbridejc @rleh @chris-durand @kikass13 @ASMfreaK

**Comment by kikass13** [comment]
👍

**Comment by salkinium** [comment]
I think we're going to use this forum, I'll start moving some of our pinned issues here and clean up the issue list to be more… issue-y.

**Reply by salkinium** [reply]
Ok, I've aggressively converted all issues that are not bugs into discussions and I feel much better about the issue list now. I'll update the documentation to point to the discussion forum instead of the issues with the Q3 release.

**Reply by twasilczyk** [reply]
I'm not so sure about moving Feature Requests from "issues" to "discussion" (e.g. #689). But this may be just myself who got used to having this kind of structure at work. On the other hand, FRs for a project like this may be difficult to track.

**Reply by salkinium** [reply]
I know, I was hesitant too. However, I've noticed the issue list getting full with feature requests that nobody is working on, so that's not great either… We'll see how this works out especially with the documentation pushing the forums more. I'm hoping for a lower hesitation to ask questions and then being table to distill issues out of that.

**Comment by TomSaw** [comment]
Yea it's good. At least it adds another layer of 'structure' :sweat_smile:
